.gitignore

```diff
@@ -5,3 +5,47 @@ packages/*/public/*.bundle.*
 packages/*/public/*.worker.*
 .DS_Store
 *-debug.log
+
+# OSX
+#
+.DS_Store
+
+# Xcode
+#
+build/
+*.pbxuser
+!default.pbxuser
```

Comment:
i have no idea, TBH. I think this is just github's default "ios project" .gitignore template